raster/r.kappa/main.c

```diff
@@ -113,9 +115,18 @@ int main(int argc, char **argv)
     flags.m->description = _("Print Matrix only");
     flags.m->guisection = _("Output settings");
 
+    flags.j = G_define_flag();
+    flags.j->key = 'j';
+    flags.j->description = _("Print in JSON");
+    flags.j->guisection = _("Output settings");
```

Comment:
For all the new JSON outputs we are adding an option right now. While flag is a nice quick hack, it is not scalable. v.db.select had two existing outputs and adding both JSON and CSV made sense there. That would be 3 flags. Now it has format which captures all the output styles. The format option is now also in t.rast.list, db.univar, and v.db.univar. r.geomorphon has profileformat with the same logic. It also fits with tools like r.out.gdal or v.in.ascii.

Reply:
+1 for unification. Done.